ui/src/app/components/help-sidebar.tsx

```diff
@@ -539,49 +552,112 @@ export const HelpSidebar = fp.flow(
       return <FlexRow style={{height: '100%', alignItems: 'center', justifyContent: 'space-around'}}>
         <img data-test-id={'help-sidebar-icon-' + icon.id} src={proIcons[icon.id]} style={{...icon.style, position: 'absolute'}} />
-        <FlexRow data-test-id='runtime-status-icon-container' style={styles.runtimeStatusIconContainer}>
+        <FlexRow data-test-id='runtime-status-icon-container' style={styles.statusIconContainer}>
           {(status === RuntimeStatus.Creating
           || status === RuntimeStatus.Starting
           || status === RuntimeStatus.Updating)
             && <FontAwesomeIcon icon={faSyncAlt} style={{
               ...styles.runtimeStatusIcon,
               ...styles.rotate,
-              color: colors.runtimeStatus.starting,
+              color: colors.asyncOperationStatus.starting,
             }}/>
           }
           {status === RuntimeStatus.Stopped
             && <FontAwesomeIcon icon={faCircle} style={{
               ...styles.runtimeStatusIcon,
               ...styles.runtimeStatusIconOutline,
-              color: colors.runtimeStatus.stopped,
+              color: colors.asyncOperationStatus.stopped,
             }}/>
           }
           {status === RuntimeStatus.Running
             && <FontAwesomeIcon icon={faCircle} style={{
               ...styles.runtimeStatusIcon,
               ...styles.runtimeStatusIconOutline,
-              color: colors.runtimeStatus.running,
+              color: colors.asyncOperationStatus.running,
             }}/>
           }
           {(status === RuntimeStatus.Stopping
           || status === RuntimeStatus.Deleting)
             && <FontAwesomeIcon icon={faSyncAlt} style={{
               ...styles.runtimeStatusIcon,
               ...styles.rotate,
-              color: colors.runtimeStatus.stopping,
+              color: colors.asyncOperationStatus.stopping,
             }}/>
           }
           {status === RuntimeStatus.Error
             && <FontAwesomeIcon icon={faCircle} style={{
               ...styles.runtimeStatusIcon,
               ...styles.runtimeStatusIconOutline,
-              color: colors.runtimeStatus.error,
+              color: colors.asyncOperationStatus.error,
             }}/>
           }
         </FlexRow>
       </FlexRow>;
     }
 
+    withinPastTwentyFourHours(epoch) {
+      const completionTimeMoment = moment(epoch);
+      const twentyFourHoursAgo = moment().subtract(1, 'days');
+      return completionTimeMoment.isAfter(twentyFourHoursAgo);
+    }
+
+    displayExtractionIcon(icon: IconConfig) {
+      const {extractionJobs} = this.state;
+      const jobsByStatus = fp.groupBy('status', extractionJobs);
+      let status;
+      // If any jobs are currently running, show running icon.
+      if (jobsByStatus[TerraJobStatus.RUNNING]) {
+        status = TerraJobStatus.RUNNING;
+      } else if (jobsByStatus[TerraJobStatus.ABORTING]) {
+        status = TerraJobStatus.ABORTING;
+      } else if (
```

Comment:
Not sure if this is ready for re-review yet, but to clarify the change I'm talking about in the above comment - these last two clauses haven't changed, here is the logic I was expecting:

> - filter jobs down to past 24h, then sort by completion time:
>   - most recent job passed: green
>   - most recent job failed: red
>   - no jobs (within 24h): no icon

Reply:
it is not but also I didn't read the thread / ticket carefully enough. easy enough to switch to this logic though.